beacon_node/store/src/forwards_iter.rs

```diff
@@ -158,6 +158,8 @@ impl<E: EthSpec, Hot: ItemStore<E>, Cold: ItemStore<E>> Iterator
             return None;
         }
         self.inner
+            .as_mut()
+            .ok()?
```

Comment:
I think we should put this `?` in the `::new` function where we call `iter_column_from`?